src/wagtail_2fa/templates/wagtail_2fa/device_list.html

```diff
@@ -24,14 +24,29 @@ <h2>
                                 </h2>
                             </td>
                             <td>
+                                {# Admins can remove existing devices from another user's account. #}
                                 <a href="{% url 'wagtail_2fa_device_remove' pk=device.pk %}" class="button">{% trans 'Remove device' %}</a>
-                                <a href="{% url 'wagtail_2fa_device_update' pk=device.pk %}" class="button">{% trans 'Update device' %}</a>
+
+                                {# Admins cannot update existing devices on another user's account. #}
+                                {% if request.user.id == device.user_id %}
+                                    <a href="{% url 'wagtail_2fa_device_update' pk=device.pk %}" class="button">{% trans 'Update device' %}</a>
+                                {% endif %}
+                            </td>
+                        </tr>
+                    {% empty %}
+
+                        <tr>
+                            <td colspan="2">
+                                There are no devices for this user.
```

Comment:
Can you make this string translatable?

Reply:
Good catch. We use translations heavily at Eventbrite.